File: 백준/Bronze/1546. 평균/평균.cc
	#define _CRT_SECURE_NO_WARNINGS
	#include <stdio.h>
	int main(void) {
		double N[1000] = { 0 };//과목 방
		int V;//과목 수
		double max = 0;
		double sum = 0 ;

		scanf("%d", &V);//과목 개수 받기

		for (int i = 0; i < V; i++) {
			scanf("%lf", &N[i]);// 과목 점수 입력
		}
		max = N[0];

		for (int i = 0; i < V; i++) {

			if (N[i] > max) {
				max = N[i];
			}
		}
	
		//printf("%lf\n", max);

		for (int i = 0; i < V; i++) {
			sum += (N[i] / max) * 100;
		}

		sum = sum / V;

		printf("%lf", sum);


	}

File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        int ave = input.nextInt();

        double sum = 0;
        double max = 0;

        double[] arr = new double[ave];

        for (int i = 0; i < ave; i++) {
            int num1 = input.nextInt();
            arr[i] = num1;
        }// ave값만큼 점수 받기


        for (int i = 0; i < ave; i++) {

            if (max < arr[i]) {
                max = arr[i];
            }
        }//점수 중 최댓값 구함

        for (int i = 0; i < ave; i++) {
            arr[i] = arr[i] / max * 100;
        }// 점수 조작


        for (int i = 0; i < ave; i++) {
            sum += arr[i];
        }

        System.out.println(sum / ave);

    }


}